fastn-js/js/ftd.js

```diff
@@ -244,7 +247,14 @@ const ftd = (function () {
             }
         }
 
-        init.body = JSON.stringify(request_json);
+        if (method === "POST") {
+            init.body = JSON.stringify(request_json);
+        }
+
+        if (method === "GET") {
+            const params = new URLSearchParams(request_json);
+            url += `?${params.toString()}`;
```

Comment:
This wont work if URL already contains some query params. Ideally we should preserve them and add these params.

Reply:
good catch, thanks!

Reply:
fixed in https://github.com/fastn-stack/fastn/pull/1813/commits/c86a7c7cc5dc7dfb59c08e6c153bfe34a52856eb